### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by thorium-zero**

### CODE REVIEW

**Review by oleksiikolii — APPROVED** [review]
I'm not sure you need to add .gitignore but code looks good

**Review by LiudmylaKulinchenko — APPROVED** [review]
Well done 🐇🍬

File: main.py [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    Genre.objects.bulk_create([Genre(name=genre) for genre in genres])
+    Actor.objects.bulk_create([
```
Comment (LiudmylaKulinchenko) [inline main.py:18]: You can skip creating the intermediate lists and directly pass the generator expressions to the bulk_create() method

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by Oleg-s-s — APPROVED** [review]
:+1:

**Pull request closed without merging**